WordPress/Classes/Utility/WPFeedbackGenerators.swift

```diff
@@ -0,0 +1,75 @@
+import UIKit
+
+@objc
+public enum WPNotificationFeedbackType : Int {
+    case Success
```

Comment:
Using the Swift 2.3 casing style here means we'll have to migrate manually to Swift 3 style, right?